palace/linalg/vector.hpp

```diff
@@ -139,6 +139,9 @@ void SetRandomReal(MPI_Comm comm, VecType &x, int seed = 0);
 template <typename VecType>
 void SetRandomSign(MPI_Comm comm, VecType &x, int seed = 0);
 
+// Component-wise square root.
+void CwiseSqrt(Vector &x);
```

Comment:
I think `Sqrt` is better, in line with the similar method from PETSc: https://petsc.org/main/manualpages/Vec/VecSqrtAbs/.